portfolio/src/main/webapp/style.css

```diff
@@ -98,6 +87,21 @@ a:hover {
   justify-content: center;
 }
 
+/* style buttons */
+#close-button {
+  height: 4vh;
```

Comment:
Why is all heights vh/vw, if the height/width of viewport changes, do we expect the size of a button to change?

Reply:
Yes! I want it to change. 😅

Reply:
That makes me wondering the ux behaviour of this page. IMO this page should be scrollable, and in the case user resizes the page either in width or height, the page should be still be relatively usable and users can still scroll to their expected content. However, with the current behaviour, this may lead to accessibility issues. E.g, a button becomes too small and untouchable at certain viewport width/height

Reply:
Oh. This page is actually not scrollable. It is a dropdown menu that is meant to fill the entire screen because there are very few elements on this dropdown menu. 
I see where you are coming from. I have resized the page and programmed it using 'responsive design mode'. The only time it fails is when the screen is unreasonably small but I intentionally chose to neglect this case because I don't have the time to do so now. When I find the time, I could definitely make it scrollable and non-responsive when the window is extremely small.  Thanks for brining this up!

Reply:
I would let is pass for now to not block you any further, but still I'm not a fan of fixed size. In a responsive design, it should be layout that changes dynamically, and children of the layout should adjust according to layout, not the viewport, that's something beyond the knowledge of itself.

Reply:
I see. Thanks for sharing! I'll take note of this in the future. (: